src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -101,7 +101,7 @@ private String getInputWithDates(String sinceDate, String untilDate) {
      * by the {@code ArgumentParser} hence this is to signal to the caller that the program is safe to exit.
      */
     private void generateReport(String inputDates)
-            throws IOException, InvalidCsvException, URISyntaxException, ParseException, HelpScreenException {
+            throws Exception {
```

Comment:
Let's move the `throws Exception` onto the previous line, with `private void generateReport..` ?